.eleventy.js

```diff
@@ -36,6 +36,7 @@ module.exports = function (config) {
 
   config.addPassthroughCopy({'./assets/_common/_img/favicons/favicon.ico': './favicon.ico' });
   config.addPassthroughCopy({'./assets/_common/_img/favicons': './img/favicons' });
+  config.addPassthroughCopy({'./assets/_common/styles': 'scss' });
```

Comment:
Not sure if this is needed because we don't use it in ospo-guide. Is this from when you were troubleshooting the CSS/Sass issues when it wasn't working on your end a while back?